crates/rolldown_plugin_transform/src/lib.rs

```diff
@@ -58,16 +59,20 @@ impl Plugin for TransformPlugin {
         return Err(anyhow::format_err!("Error occurred when parsing {}\n: {:?}", args.id, errs));
       }
     };
+
+    let env = if self.target.is_some() && self.browserslist.is_some() {
+      Err("Cannot specify both `target` and `browserslist` at the same time".to_string())
```

Comment:
Is this required by oxc-transfomer? For better UX, I guess we could support accepting  both `target` and `browserslist`.

Reply:
At this point, I'm guessing the Oxc API isn't ready yet.